Presentation.Web/Controllers/API/BaseApiController.cs

```diff
@@ -190,7 +200,60 @@ protected virtual IQueryable<T> Page<T>(IQueryable<T> query, PagingModel<T> pagi
                                                                 Newtonsoft.Json.JsonConvert.SerializeObject(
                                                                     paginationHeader));
 
-            return query.OrderByField(paging.OrderBy, paging.Descending).Skip(paging.Skip).Take(paging.Take);
+            //Make sure query is ordered
+            query = query.OrderByField(paging.OrderBy, paging.Descending);
+
+            //Apply post-processing
+            query = paging.ApplyPostProcessing(query);
+
+            //Load the page
+            return query
+                .Skip(paging.Skip)
+                .Take(paging.Take);
+        }
+
+        #region access control
+        protected bool AllowOrganizationAccess(int organizationId)
+        {
+            if (ApplyNewAccessControlScheme())
```

Comment:
Bare en tanke, men måske kan if-tingen blive til en Strategy, hvis det skal være rigtig pænt og så kan hele Allow funktionaliteten blive en helper

Reply:
Jo det skal være to strategies. Det giver mening.